tests/integration/api_v2/http/transactions.test.js

```diff
@@ -110,13 +110,13 @@ describe('Transactions API', () => {
 		});
 
 		it('long transaction id -> 500', async () => {
-			const response = await api.get(`${endpoint}?id=412875216073141752800000`, 500);
+			const response = await api.get(`${endpoint}?transactionId=412875216073141752800000`, 500);
 			expect(response).toMap(badRequestSchema);
 		});
 
 		it('invalid transaction id -> 500', async () => {
-			const response = await api.get(`${endpoint}?id=41287`, 500);
-			expect(response).toMap(notFoundSchema);
+			const response = await api.get(`${endpoint}?transactionId=41287`, 500);
```

Comment:
The returned error code should be `404 NOT FOUND`.

Reply:
Added code to handle this error scenario.